topics/admin/tutorials/toolshed/slides.html

```diff
@@ -0,0 +1,224 @@
+---
+layout: base_slides
+logo: "GTN"
+
+title: "The Galaxy Tool Shed"
```

Comment:
We already have https://galaxyproject.github.io/training-material/topics/dev/tutorials/toolshed/slides.html , not sure if these slides are needed.

Reply:
@martenson what do you want to do?

Reply:
I plan to merge them in for now, we can reconvene and merge the materials after the training is over. That 80 slide deck is not something easy to grasp or follow, I'd rather have smaller one-purpose decks in the future.